diff --git a/dist/rollupAdapter.js b/dist/rollupAdapter.js
index 60d97f7d6689841632e8278d81f8816ce68da2ee..81f7e1a4c375e80527f80cd663003c3ed6034bae 100644
--- a/dist/rollupAdapter.js
+++ b/dist/rollupAdapter.js
@@ -75,7 +75,8 @@ function rollupAdapter(rollupPlugin, rollupInputOptions = {}, adapterOptions = {
             if (!injectedFilePath && !rollupPlugin.resolveId) {
                 return;
             }
-            if (!injectedFilePath && !path_1.default.isAbsolute(source) && whatwg_url_1.default.parseURL(source) != null) {
+            if (!source.startsWith('\0') && !injectedFilePath && !path_1.default.isAbsolute(source) && whatwg_url_1.default.parseURL(source) != null) {
+            // if (!injectedFilePath && !path_1.default.isAbsolute(source) && whatwg_url_1.default.parseURL(source) != null) {
                 // don't resolve relative and valid urls
                 return source;
             }
